CsvParser/csvParser.go

```diff
@@ -1,18 +1,20 @@
 package csvParser
 
-import(
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
 	"io"
-	"time"
-  	"log"
+	"log"
 	"os"
 	"strconv"
-	"encoding/csv"
 	"strings"
+	"time"
+
```

Comment:
Is there a reason you left this line blank? I would get rid of it

Reply:
Blank line removed